components/style/properties/gecko.mako.rs

```diff
@@ -3151,7 +3141,7 @@ fn static_assert() {
             },
             _ => None,
         };
-        longhands::transform::computed_value::T(result)
+        Transform(result.unwrap_or(vec!()))
```

Comment:
Please fixup the function above to directly create a `vec![]`, this looks kinda weird:

```rust
let result = if !vec.is_empty() { Some(vec) } else { None };
Transform(result.unwrap_or(vec![]))
```